src/Icons/src/Command/WarmCacheCommand.php

```diff
@@ -41,16 +39,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $io = new SymfonyStyle($input, $output);
         $io->comment('Warming the icon cache...');
 
-        foreach ($this->icons->icons() as $icon) {
-            try {
-                $this->registry->get($icon, refresh: true);
-
-                if ($output->isVerbose()) {
-                    $io->writeln(sprintf(' Warmed icon <comment>%s</comment>.', $icon));
+        $this->warmer->warm(
+            onSuccess: function (string $name) use ($io) {
+                if ($io->isVerbose()) {
+                    $io->writeln(sprintf(' Warmed icon <comment>%s</comment>.', $name));
```

Comment:
Minor comment: could we replace _"Warmed icon ..."_ by something that describes with more precision what's going on? E.g. _"Downloaded icon ..."_ or _"Extracted icon ..."_

Reply:
Hmm, with the currently architecture, in this spot, I can't tell exactly what happened (if it was downloaded or just extracted from a local svg). Adding verbosity to the command I think could get the http-client logs.